### ISSUE TITLE
Package is abandoned!?

### ISSUE THREAD

**Issue by christianpv** [Chumper/Datatable#359]
The package has been marked as abandoned on packagist and a replacement was not suggested but I think it would be wise to let people know about https://github.com/OpenSkill/Datatable while mentioning that it's on development.

**Comment by Chumper** [comment]
Yes, maybe I was a little bit too fast with the abandoned button. 
I just wanted to say that I will not update my personal package (Chumper/Datatable) anymore and we are working on a new version. I will it as a suggestion and make clear that this package will not receive any new updates.

**Comment by christianpv** [comment]
Thanks for the quick response. Are there any major differences between the two? When do you think the other package will be somewhat stable?

**Comment by Chumper** [comment]
There will be major differences between the two packages.
The main idea will be the same, but syntactically it will be different. 
Much more decoupled and testable.

I want to release an early development version this weekend with only `Datatables 1.9` support and the `CollectionProvider`.

I am aiming to release a stable version with 
* `Datatables 1.9` 
* `Datatables 1.10` 
* `Dynatable 0.3.1`

as well as 
*`CollectionProvider`
*`EloquentModelProvider` 
*`EloquentQueryProvider` 

in late January, but i am sure there will be a few version in between.

**Comment by christianpv** [comment]
Thanks for the detailed response. When you manage to release the early dev version let me know and I will see if I can test it on my dev project as I'm using the package for Datatable 1.9 and only the CollectionProvider.

**Comment by Chumper** [comment]
Ping!

I just released version `0.1.0` and `0.1.1` shortly after.
I still have a lot of stuff to document, but the `CollectionProvider` should already work quite well.

**Comment by younes0** [comment]
:+1:
